src/main/java/org/spring/FullVersion/FamilyService.java

```diff
@@ -70,14 +69,18 @@ public Family bornChild(Family family, String masculine, String feminine) {
     }
 
     public Family adoptChild(Family family, Human child) {
+        Human father = family.getFather();
+        child.setSurname(father.getSurname());
+        child.setIq(father.getIq());
```

Comment:
this should be in bornChild method not here, on this method just add the received child